javascript/src/main/utils/RoomSlotFormHelpers.js

```diff
@@ -0,0 +1,24 @@
+// Takes a time in a format like XX:XXAM and converts it to HH:MM:SS format
+// to be turned into a LocalTime.
+const formatTimeToLocalTime = (time) => {
```

Comment:
I don't think this needs to be addressed in this PR but it should be noted that this logic should be replaced with a proper date parsing/formatting library in a future refactoring story.